src/main/scala/firrtl/WIR.scala

```diff
@@ -215,32 +215,43 @@ case class ExpWidth(arg1: Width) extends Width with HasMapWidth {
 object WrappedType {
   def apply(t: Type) = new WrappedType(t)
   def wt(t: Type) = apply(t)
-}
-class WrappedType(val t: Type) {
-  def wt(tx: Type) = new WrappedType(tx)
-  override def equals(o: Any): Boolean = o match {
-    case (t2: WrappedType) => (t, t2.t) match {
+  private def compare(strict: Boolean)(t1: Type, t2: Type): Boolean =
```

Comment:
Perhaps rename the arguments from t1/t2 to source/sink. This way it implies that the argument order is important. I'd also add comments explaining the arguments.

Reply:
done